examples/blog/index.p6

```diff
@@ -21,20 +25,24 @@ model Person is rw {
 }
 
 my $*RED-DEBUG = True;
-for database("Pg"), database("SQLite") -> $*RED-DB {
+my $*RED-DB = database("SQLite");
 
-    Person.^create-table;
-    Post.^create-table;
+say "✓ Creating tables for Person and Post";
+Person.^create-table;
+Post.^create-table;
 
-    my $p  = Person.^create: :name<Fernando>;
+say "✓ Creating a Person";
+my $p  = Person.^create: :name<Fernando>;
 
-    my $post = $p.posts.create: :title<Bla>, :body<BlaBle1>;
+say "✓ Creating a blog Post";
+my $post = $p.posts.create: :title<Bla>, :body<BlaBle1>, :author<Fernando>;
```

Comment:
the `:author<Fernando>` shouldn't be needed, right? its being created by the `$p`'s posts, so it should populate that